scripts/build-post-list.js

```diff
@@ -58,10 +59,25 @@ function walkDirectories(directories, result, sectionWeight = 0, sectionTitle) {
         details.isIndex = fileName.endsWith('/index.md')
         details.slug = details.isIndex ? sectionSlug : slug.replace(/\.md$/, '')
         if(details.slug.includes('/specifications/') && !details.title) {
-          if(basename(data.slug).startsWith('v')) {
-            details.title = capitalize(basename(data.slug).slice(1))
+          let fileName = basename(data.slug)
+
+          if(fileName.includes('release')) {
+            details.isPrerelease = true
+            details.releaseDate = getReleaseDate(fileName)
+            fileName = fileName.split('-')[0]
```

Comment:
it feels not right to mutate variable like this, especially that `fileName` is initialized not as file name really. can't we do it all in one place at line 62? best with a comment as you did for `getReleaseDate` that explains the input example and output?